docs/references/api.md

```diff
@@ -0,0 +1,297 @@
+# Pro API Reference Guide
+
+The Pro Client has a Python-based API to be consumed by users who want to integrate the client's functionality to their software.
+The functions and objects are available through the `uaclient.api` module, and all of the available endpoints return an object with specific data for the calls.
+
+Besides importing the Python code directly, consumers who are not writing Python may use the CLI to call the same functionality, using the `pro api` command. This command will always return a JSON with a standard structure, as can be seen below:
+
+```json
+{
+    "_schema_version": "v1",
+    "data": {
+        "attributes": {
+            <endpoint specific attributes>
```

Comment:
I think we can add "_"  instead of white spaces just so we don't have that highlight on the page

Reply:
Actually made it a command, so it does not break syntax